flink-jepsen/project.clj

```diff
@@ -19,10 +19,11 @@
             :url  "http://www.apache.org/licenses/LICENSE-2.0"}
   :main jepsen.flink.flink
   :aot [jepsen.flink.flink]
-  :dependencies [[org.clojure/clojure "1.9.0"],
+  :dependencies [;; jepsen 0.1.19 is not compatible with 1.10.1+
+                 [org.clojure/clojure "1.10.0"],
                  [cheshire "5.8.0"]
                  [clj-http "3.8.0"]
-                 [jepsen "0.1.13"],
+                 [jepsen "0.1.19"],
```

Comment:
`0.2.5` is probably not working, right?

Reply:
I didn't try; 1.19 is the last release in the 1.x series, and I didn't wanna deal with any broken APIs :/

Reply:
Fair enough.